episodes/30-dplyr.Rmd

```diff
@@ -938,7 +938,7 @@ by clicking on the link and then moving it to the `data/` folder, or
 you can use the R code below to download it directly to the folder.
 
 ```{r, message=FALSE}
-download.file(url = "https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/_episodes_rmd/data/annot1.csv",
+download.file(url = "https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/episodes/data/annot1.csv",
```

Comment:
I would suggest to use the URL for the lesson website, but this is entirely your preference. 

```suggestion
download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.csv",
```

Reply:
I don't think this works:

Reply:
While
```
> download.file(url = "https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/episodes/data/annot1.csv", destfile = "annot1.csv")
trying URL 'https://raw.githubusercontent.com/carpentries-incubator/bioc-intro/main/episodes/data/annot1.csv'
Content type 'text/plain; charset=utf-8' length 826 bytes
==================================================
downloaded 826 bytes
```

Reply:
> I don't think this works:
> 
> ```
> > download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs", destfile = "annot1.csv")
> trying URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs'
> Error in download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs",  : 
>   cannot open URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs'
> In addition: Warning messages:
> 1: In download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs",  :
>   downloaded length 0 != reported length 9339
> 2: In download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs",  :
>   cannot open URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.cs': HTTP status was '404 Not Found'
> ```

There is a typo in your URL. It should end in "csv":


```r
> download.file(url = "https://carpentries-incubator.github.io/bioc-intro/data/annot1.csv", destfile = "annot1.csv")                                                          
trying URL 'https://carpentries-incubator.github.io/bioc-intro/data/annot1.csv'
Content type 'text/csv; charset=utf-8' length 826 bytes
==================================================
downloaded 826 bytes
```